locales/en/pages/augustrecess.yml

```diff
@@ -0,0 +1,123 @@
+# Twitter Card Validator (https://cards-dev.twitter.com/validator) after changing
+# these values.
+sharing:
+  title: Join the People’s Lobby for Net Neutrality
+  description: |
+    Join the People’s Lobby for Net Neutrality to meet with your lawmakers
+    during the August congressional recess
+  image: https://www.battleforthenet.com/images/share_images/august-recess-social.jpg
+  url: https://www.battleforthenet.com/augustrecess
+
+title: Join the People’s Lobby for Net Neutrality
+intro_html: |
+  The antidote to Big Cable’s campaign to destroy the open Internet is real,
+  everyday people joining forces to hold Congress accountable. That’s why we’re
+  organizing and training people passionate about net neutrality to meet with
+  their representatives in-district during August recess. The dates and times
+  of the meetings are still being set. If you're interested in attending, please
+  fill-in the information below.
+cta_button: Sign up
+tweet_text: |
+  Join the People’s Lobby for Net Neutrality to meet with your lawmakers during
+  the August congressional recess
+
+form:
+  title: "Sign Up"
+  description_html: |
+    <p>
+      We’re working to set up meetings with lawmakers around the country. The
+      dates and times are still TBD, so if you're interested in attending,
+      please fill in the information below.  We’ll get back to you soon with
+      more details.
+    </p>
+  success: You’ve successfully signed up! Check out the Facebook groups below.
+  generic_error: That didn't work for some reason 😞
+  firstname_placeholder: First name*
+  lastname_placeholder: Last name*
+  email_placeholder: Email*
+  address_placeholder: Address*
+  zip_placeholder: ZIP*
+  phone_placeholder: Phone*
+  comments_placeholder: Why does net neutrality matter to you?
+  meeting_question: |
+    Can you attend a meeting that is scheduled between business hours
+    Monday-Friday?
+  business_owner_question: Are you a small business owner?
+  veteran_question: Are you a veteran?
+  button_loading: Saving...
+  button_cta: Sign up
+  yes: Yes
+  no: No
+  disclaimer_html: |
+    <a target="_blank" href="https://www.fightforthefuture.org">
+      Fight for the Future</a>,
+    <a target="_blank" href="https://www.demandprogress.org">
+      Demand Progress</a>,
+    and <a href="https://www.freepress.net">Free Press Action Fund</a>
+    will email you updates, and you can unsubscribe at any time. If you enter
+    your number (it’s optional) we will follow up by SMS. Message &amp; data
+    rates apply. You can always text STOP to stop receiving messages.
+    <a target="_blank" href="/privacy">Privacy Policy</a>
+
+# New sections created here will be added dynamically to the page in the order
+# they're listed. The section key (website, social, etc.) will be used to create
+# an anchor link (social = https://battleforthenet.com/augustrecess#social)
+sections:
+  what:
+    title: What’s happening?
+    body_html: |
```

Comment:
The `_html` keys get passed through a markdown parser, so you can skip the `<p>` tags here. (Fine to leave them in, though.)

Reply:
I added them because I noticed in one particular instance the `<p>` tags did not get added (but in the next section they did). Why I'm not sure. Maybe it even had something to do with hot reloading or strange state, but I put them in everywhere for safety since the styles and margins are a little different without them.